### PR TITLE
Fix wrong parsing of ktlint version.

### LINKED ISSUE #28: Unable to specify any reporter if ktlint version is bigger than "0.9.x"
There seems to be a bug that prevents you from using a custom reporter when ktlint version is bigger than "0.9.x", eg. "0.10.1".

This is due to comparing strings instead of integers:

```
private fun isReportAvailable(version: String, availableSinceVersion: String): Boolean {
    val versionsNumbers = version.split('.')
    val reporterVersionNumbers = availableSinceVersion.split('.')
    return versionsNumbers[0] >= reporterVersionNumbers[0] &&
            versionsNumbers[1] >= reporterVersionNumbers[1] &&
            versionsNumbers[2] >= reporterVersionNumbers[2]
}
```
To reproduce the issue you can use this configuration:
```
ktlint {
    version = "0.10.1"
    debug = true
    verbose = false
    reporter = "json"
    ignoreFailures = true
}
```
The output is:
```
> Task :app:ktlintItalyCheck
[DEBUG] Discovered ruleset "standard"
[DEBUG] Discovered reporter "plain"
[DEBUG] Discovered reporter "json"
[DEBUG] Discovered reporter "checkstyle"
[DEBUG] Initializing "plain" reporter with {verbose=false}
[DEBUG] {} loaded from .editorconfig
[...]
```

### PR DESCRIPTION
**Pull request by Tapchicoma**
This prevents to use plugin with >0.10.x versions of ktlint.

Fixes #28 

Signed-off-by: Yahor Berdnikau <egorr.berd@gmail.com>

### CODE REVIEW

File: src/main/kotlin/org/jlleitschuh/gradle/ktlint/Reporter.kt [hunk 1]
```diff
@@ -37,8 +37,8 @@ fun JavaExec.applyReporter(target: Project, extension: KtlintExtension) {
 }
 
 private fun isReportAvailable(version: String, availableSinceVersion: String): Boolean {
-    val versionsNumbers = version.split('.')
-    val reporterVersionNumbers = availableSinceVersion.split('.')
+    val versionsNumbers = version.split('.').map { it.toInt() }
+    val reporterVersionNumbers = availableSinceVersion.split('.').map { it.toInt() }
```
Comment (JLLeitschuh) [inline src/main/kotlin/org/jlleitschuh/gradle/ktlint/Reporter.kt]: @shyiko Just want to check with you, do you think you will ever have a release that has some sort of `RC1` specifier on the end?
If not this should be fine.

Comment (Tapchicoma) [reply src/main/kotlin/org/jlleitschuh/gradle/ktlint/Reporter.kt]: @JLLeitschuh I may apply your proposal to use symver plugin in a separate PR

Comment (shyiko) [reply src/main/kotlin/org/jlleitschuh/gradle/ktlint/Reporter.kt]: @JLLeitschuh never (and you can count on it). ktlint adheres to strict semver without pre-release|s or build meta.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased]
+###
```
Comment (JLLeitschuh) [inline CHANGELOG.md]: Can you make this header `Fixed` like the one below.

Comment (Tapchicoma) [reply CHANGELOG.md]: Sorry, missed this. Fixed in updated PR.

**Comment by JLLeitschuh** [comment]
Thanks!

**Pull request merged**